.github/workflows/CI.yaml

```diff
@@ -0,0 +1,65 @@
+name: CI
+
+# Run this workflow every time a new commit pushed to your repository
+on: push
+
+jobs:
+  build-and-test:
+    name: Build and test
+    runs-on: ubuntu-latest
+
+    steps:
+      - run: sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv E52529D4
+      - run: echo "deb [ arch=amd64 ] https://repo.mongodb.org/apt/ubuntu xenial/mongodb-org/4.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-4.0.list
+      - run: sudo apt-get update
+      - run: sudo apt-get install -y gnupg
+      - run: sudo apt-get install -y mongodb-org-shell
+
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Define Gradle cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+
+      - name: Build
+        run: ./gradlew build --build-cache
+
+      - name: Upload Unit Test Results
+        if: always()
+        uses: actions/upload-artifact@v2
+        with:
+          name: tests-results # Name artifact for storage in cache
+          path: |
+            modules/cli/build/test-results/**/*.xml
+            modules/core/build/test-results/**/*.xml
+            modules/driver-mongo/build/test-results/**/*.xml
+            modules/test/build/test-results/**/*.xml
```

Comment:
I am afraid that tests results for new modules will be forgotten if paths are declared like that. Wouldn't that be possible to declare a unique path like that: ```modules/**/build/test-results/**/*.xml```